Dockerfile.virtual

```
@@ -1,14 +1,14 @@
 # Build
-FROM mcr.microsoft.com/ccf/app/dev:3.0.0-dev6-sgx as builder
+FROM mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-sgx as builder
```

```suggestion
FROM mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-virtual as builder
```

New containers for this release too.